_includes/people/perez-suarez.david.html

```diff
@@ -2,8 +2,9 @@
   <img class="media-object pull-left" src="{{page.root}}/img/people/perez-suarez.david.jpg" alt="David P&eacute;rez-Su&aacute;rez" />
   <div id="perez-suarez.david" class="media-body">
     <a href="http://dpshelio.github.io/" class="person">David P&eacute;rez-Su&aacute;rez</a>
-    is a post-doc at the Space Weather Team in the South African National Space Agency (SANSA), 
-    working as the Solar Physicist of the team. 
+    is a post-doc at the <a href="http://mssl.ucl.ac.uk/">Mullard Space Science Laboratory</a>
+    (<a href="http://www.ucl.ac.uk/">University College London </a>),
+    as a member of the Solar Physics group.
     He mainly works doing image processing and time-series analisys of solar data, 
     but he is also active in a couple of projects involving webservices, workflows and grid computing.
     David is one of the developers of <a href="http://sunpy.org">a python library for solar physicist</a> 
```

Comment:
`Python` instead of `python` (title case)
"solar physicists" instead of "solar physicist" (plural instead of singular).